greengrass-ipc-model/main.smithy

```diff
@@ -192,21 +193,22 @@ operation SubscribeToIoTCore {
     errors: [ServiceError, UnauthorizedError]
 }
 
-// This API can be used only by stream manager, customer component calling this API will receive UnauthorizedError
+/// Validate authorization token
+/// NOTE This API can be used only by stream manager, customer component calling this API will receive UnauthorizedError
 operation ValidateAuthorizationToken {
     input: ValidateAuthorizationTokenRequest,
     output: ValidateAuthorizationTokenResponse,
     errors: [InvalidTokenError, UnauthorizedError, ServiceError]
 }
 
-// Retrieves a secret stored in AWS secrets manager
+/// Retrieves a secret stored in AWS secrets manager
 operation GetSecretValue {
     input: GetSecretValueRequest,
     output: GetSecretValueResponse,
     errors: [UnauthorizedError, ResourceNotFoundError, ServiceError]
 }
 
-// Generate a password for the HttpDebugView component
+/// Generate a password for the HttpDebugView component
```

Comment:
LocalDebugConsole

Reply:
To where `LocalDebugConsole ` should be added?

Reply:
It isn't called HttpDebugView. It is LocalDebugConsole